src/Sulu/Component/Webspace/Analyzer/WebsiteRequestAnalyzer.php

```diff
@@ -398,7 +398,7 @@ private function getResourceLocatorFromRequest(
 
         $path = rtrim(implode('/', $pathParts), '/') . '/' . $fileInfo[0];
         if (count($fileInfo) > 1) {
-            $formatResult = $fileInfo[1];
+            $formatResult = end($fileInfo);
```

Comment:
hmm your completly right but i think of releasing this in a HOTFIX ... @danrot what did you think?